File: C++ Language/Q8.cpp
#include <iostream>
using namespace std;

int main() {
    float fahrenheit, celsius;

    cout << "Enter temperature in Fahrenheit: ";
    cin >> fahrenheit;

    // Formula: C = 5/9 * (F - 32)
    // We use 5.0 / 9.0 so the program knows we want decimal results
    celsius = (5.0 / 9.0) * (fahrenheit - 32);

    cout << "Temperature in Celsius: " << celsius << endl;

    return 0;
}

File: Python Language/Q8.py
# Program to convert temperature from Fahrenheit to Celsius

# Getting temperature in Fahrenheit
f = float(input("Enter temperature in Fahrenheit: "))

# Converting to Celsius using the formula C = 5/9 * (f - 32)
c = (5/9) * (f - 32)

# Printing the result
print(f"Temperature in Celsius: {c:.2f}")